velox/functions/sparksql/tests/StringTest.cpp

```diff
@@ -266,6 +266,22 @@ TEST_F(StringTest, left) {
   EXPECT_EQ(left("da\u6570\u636Eta", 30), "da\u6570\u636Eta");
 }
 
+TEST_F(StringTest, right) {
+  const auto right = [&](const std::optional<std::string>& str,
+                        const std::optional<int32_t>& length) {
+    return evaluateOnce<std::string>("right(c0, c1)", str, length);
+  };
+  EXPECT_EQ(right("example", -2), "");
+  EXPECT_EQ(right("example", 0), "");
+  EXPECT_EQ(right("example", 2), "le");
+  EXPECT_EQ(right("example", 7), "example");
+  EXPECT_EQ(right("example", 20), "example");
```

Comment:
Could we add test for empty input string like `right("", 20)`?

Reply:
Empty string test is returned.